bindings/python/linkgrammar.py

```diff
@@ -29,7 +29,13 @@ def __init__(self, verbosity=0,
                  spell_guess=False,
                  use_sat=False,
                  max_parse_time=-1,
-                 disjunct_cost=2.7):
+                 disjunct_cost=2.7,
+            #     debug="",
+            #     test="",
+                 use_viterbi=False,
+                 use_cluster_disjuncts=False,
```

Comment:
May it be just commented out or for mo clearance of understanding be removed ad all.
Should I do it for interface file link_grammar.i or it could be as is?